NOTICE-THIRD-PARTY.md

```diff
@@ -245,10 +245,10 @@ Protocol Buffers [Core] (3.8.0)
  * Source: https://github.com/protocolbuffers/protobuf/protobuf-java
 
 
-SLF4J API Module (1.7.30)
+SLF4J API Module (1.7.25)
 
  * License: MIT License
- * Maven artifact: `org.slf4j:slf4j-api:1.7.30`
+ * Maven artifact: `org.slf4j:slf4j-api:1.7.25`
```

Comment:
Actually the version has not changed, but somehow the Maven plugin which generates this file grabs the wrong version now. I could not figure out how to fix this and I leave that for now as it is.

Reply:
In the final bundle and during runtime, the correct version 1.7.30 is still used.